ee/clickhouse/queries/test/test_cohort_query.py

```diff
@@ -2216,7 +2216,7 @@ def test_performed_event_sequence_and_clause_with_additional_event(self):
         )
 
         q, params = CohortQuery(filter=filter, team=self.team).get_query()
-        res = sync_execute(q, params)
+        res = sync_execute(q, {**params, **filter.hogql_context.values})
```

Comment:
This is a smell - params handling is already a major painpoint and I'd be careful about making it worse. Why doesn't CohortQuery do this itself?